### ISSUE TITLE
Screen Reader(NVDA) focus is not moving further after reaching Typeahead (Auto-Complete) field, while navigating using down arrow and also unable to change the mode when the focus is on Typeahead field.

### ISSUE THREAD

**Issue by anitha01501** [nvaccess/nvda#13140]
[###](url) Steps to [reproduce:](url)
<img width="471" alt="Typeahead field Issue" src="https://user-images.githubusercontent.com/68206996/144973582-817289e9-1e7b-4438-95a9-b8583613c14d.png">

Step 1: Enable Screen Reader(NVDA) and Navigate to the App
Step 2: Click on the Filter popup button which expands the filter section with multiple fields and focus moves automatically to the typeahead field.
Step 3: Try to navigate away from the typeahead field ('what do you want to see' field ) using linear navigation (Up/Down arrow keys).

### Actual behavior:
Screen Reader(NVDA) focus is not moving further after invoking Filter button, while navigating using down arrow mode.
Ex: After invoking Filter popup button Screen Reader(NVDA) focus is moving to 'What do you want to see' list box and after pressing down arrow unable to navigate further and unable to change the mode.

### Expected behavior:
Screen Reader(NVDA) focus should move further after invoking Filter button, while navigating using down arrow mode.
Ex: After invoking Filter button Screen Reader(NVDA) focus should move to controls available in the filter section and should narrate name role and state.

**Comment by Adriani90** [comment]
@anitha01501 please provide a minimal test case, a website or a codepen where we can test this. Note that here are alot of blind developers and many people are not able to read your screenshot.

Do you still have this issue in NVDA 2023.1?

**Comment by Adriani90** [comment]
We didn't get any updates from the initial author since years, I am closing as abandoned. Please provide the required info and we can reopen.
